src/jade/tabs/trust.jade

```diff
@@ -1,19 +1,21 @@
-section.col-xs-12.content(ng-controller="TrustCtrl")
+section.col-xs-12.content(ng-controller='TrustCtrl')
 
   //- Disconnected
 
-  group.disconnected(ng-hide="connected")
+  group.disconnected(ng-hide='connected')
     p.literal(l10n) You have to be online to see this screen
 
-  group.disconnected(ng-hide="!connected || loadState.lines")
+  group.disconnected(ng-hide='!connected || loadState.lines')
     p.literal(l10n) Loading...
 
   //- Connected
 
-  group(ng-show="connected && loadState.lines")
-    div(ng-show="!loadingAccount && !account.Balance && loadState.account && connected")
+  group(ng-show='connected && loadState.lines')
+    div(ng-show='!loadingAccount && !account.Balance && loadState.account && connected')
       include banner/unfunded
 
+    // label. Contains message to the user
```

Comment:
But also notice the comment in L21 doesn't have '-' and is still interpreted as a comment.